lighthouse-core/formatters/partials/table.html

```diff
@@ -26,6 +26,12 @@
 .table_list tr:hover {
   background-color: #fafafa;
 }
+.table_list td * {
```

Comment:
Perf isnt a huge deal on the report, but `*` is always nasty. Is there something more specific we can use? Right now it's just br, code blocks as children elements. RIght?